### ISSUE TITLE
Explain how content changes when voting

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#19]
Form the October 2022 Accessibility Report ...

> When the answer is selected the question immediately changes – this fails 3.2.2; because context shouldn’t change without confirmation or warning. 

Suggest including an explaination of how the tool works - possibly as a brief standalone introduction page

**Comment by patcon** [comment]
## Screencast

[![Clicking the voting buttons on Polis a conversation](http://g.recordit.co/GuQznhdZuT.gif)](https://recordit.co/GuQznhdZuT)

## Context

Understanding WCAG 2.0: [On Input. Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

> ### Specific Benefits of Success Criterion 3.2.2:
>
> This Success Criterion helps users with disabilities by making interactive content more predictable. Unexpected changes of context can be so disorienting for users with visual disabilities or cognitive limitations that they are unable to use the content.

**Comment by patcon** [comment]
A fix for this might involve creating alerts or live regions to explain what interactions are happening elsewhere on the page:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Not sure whether these messages should be assertive (immediately read aloud) or polite (default; read after user is otherwise idle).

I could imagine that the visualization below could or should also have a live region message, though I imagine that is out-of-scope. The crucial information for the visualization might be:
- number of groups have increased/decreased
- users own avatar has moved in/out of a group
- group shapes have updated. this can happen even without user interaction, esp on regular interval when conversation is active and math worker is working away processing data.

Should we re-ticket any of the above comments? Open to suggestions!
